---
title: William Fairbrother, Hotspot Exons
---

# Hotspot exons - a thousand points of vulnerability to miss-splicing events

## William Fairbrother, Brown University

Splicing reaction occurs by nucleophilic attack of the phosphate group by -OH groups
at the 5' site followed by the 3' site. 
It is catalyzed by a large complex called the spliceosome. 
See CSH Perspectives in Biology, 3(7) 2011.

Intron removal in multi-exon contect may differ from existing model.
Multiple splicing events associated with intron removal:
(1) Concurrent splicing (Staley lab), 
(2) Recursive splicing (Javier Lopez),
(3) Stochastic splicing (Wan et al Cell 2021, Larson lab).

An average human coding gene has 9.8 exons.
Exon skipping alone could generate $2^{10}$ variations.
Alternative splicing expands human proteome.
See Lee and Rio, 2015.
Also, more recently 300K RNA-Seq experiment. 

Sequence Read Archive (SRA) reveals numerous splicing events outside the annotation. 

When a splice site is disrupted by mutation,
54% of the time the spliceosome uses the next best choice.
Dawes et. al, Nature Genetics, 55, 324-332 (2023)

MPRA finds a 10 fold excess of 5's and 3's in typical transcript.

Fairbrother lab looks at splicing defects. 
Mutations in exons can disrupt splice sites.

The strength of flanking exons affects sensitivity to splicing mutations.

  - Some diseases are enriched in splicing nutations.
  - Splicing mutations are distributed non-uniformly.
  - Exon level features are more predictive of exonic splicing mutations than local motif features. Hotspot exons have higher GBM score than non-hotspot exons.
  - SpliceAI predicted hotspot exons are more vulnerable to splicing inhibitor PlaB.
  - In addition to exon features, flanking exon strength can affect nutation.
  - Only weak exons are affected by flanking exon strengtg,
  - Test ASO oligos. ASO targeting flanking exon can restore splicng.
