<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.361">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Tajurba - 2023-04-14</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../../site_libs/quarto-nav/headroom.min.js"></script>
<script src="../../site_libs/clipboard/clipboard.min.js"></script>
<script src="../../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../../site_libs/quarto-search/fuse.min.js"></script>
<script src="../../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../../">
<link href="../../listen/group-meeting/2023-04-19-tatsuhiko-naito.html" rel="next">
<link href="../../listen/group-meeting/2023-04-12-adas-group-meeting.html" rel="prev">
<script src="../../site_libs/quarto-html/quarto.js"></script>
<script src="../../site_libs/quarto-html/popper.min.js"></script>
<script src="../../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../../site_libs/quarto-html/anchor.min.js"></script>
<link href="../../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<link rel="stylesheet" href="../../styles.css">
</head>

<body class="nav-sidebar docked nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../../index.html">
    <span class="navbar-title">Tajurba</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../../journal/index.html" rel="" target="">
 <span class="menu-text">Journal</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../read/index.html" rel="" target="">
 <span class="menu-text">Read</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../../listen/index.html" rel="" target="" aria-current="page">
 <span class="menu-text">Listen</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../think/index.html" rel="" target="">
 <span class="menu-text">Think</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../../code/index.html" rel="" target="">
 <span class="menu-text">Code</span></a>
  </li>  
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../../listen/group-meeting/2023-04-12-adas-group-meeting.html">Group Meetings</a></li><li class="breadcrumb-item"><a href="../../listen/group-meeting/2023-04-14-adsp-group-meeting.html">2023-04-14</a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
      <button type="button" class="btn quarto-search-button" aria-label="" onclick="window.quartoOpenSearch();">
        <i class="bi bi-search"></i>
      </button>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation docked overflow-auto">
        <div class="mt-2 flex-shrink-0 align-items-center">
        <div class="sidebar-search">
        <div id="quarto-search" class="" title="Search"></div>
        </div>
        </div>
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Listen</span></a>
  </div>
</li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true">
 <span class="menu-text">Group Meetings</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/group-meeting/2023-04-12-adas-group-meeting.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-04-12</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/group-meeting/2023-04-14-adsp-group-meeting.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">2023-04-14</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/group-meeting/2023-04-19-tatsuhiko-naito.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-04-19</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/group-meeting/2023-04-26-scott-adamson.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-04-26</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/group-meeting/2023-05-10-brielin-brown.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-05-10</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/group-meeting/2023-06-07-sei-chang.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-06-07</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="false">
 <span class="menu-text">Journal Clubs</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-2" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-2" class="collapse list-unstyled sidebar-section depth1 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/journal-club/2023-04-13-coop-book-chapter-6.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 6, Population Genetics by Graham Coop</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/journal-club/2023-04-27-coop-book-chapter-7.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 7, Population Genetics by Graham Coop</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/journal-club/2023-05-11-coop-book-chapter-8.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 8, Population Genetics by Graham Coop</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/journal-club/2023-05-25-coop-book-chapter-9.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 9, Population Genetics by Graham Coop</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/journal-club/2023-06-01-otters.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">OTTERS: a powerful TWAS framework leveraging summary-level reference data</span></a>
  </div>
</li>
      </ul>
  </li>
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a class="sidebar-item-text sidebar-link text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="false">
 <span class="menu-text">Seminars</span></a>
          <a class="sidebar-item-toggle text-start collapsed" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-3" aria-expanded="false" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-3" class="collapse list-unstyled sidebar-section depth1 ">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/seminar/2023-05-04-julia-domingo.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-04-26</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../../listen/seminar/2023-06-01-william-fairbrother.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">2023-04-26</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#adsp-group-meeting-at-icahn-institute" id="toc-adsp-group-meeting-at-icahn-institute" class="nav-link active" data-scroll-target="#adsp-group-meeting-at-icahn-institute">ADSP Group Meeting at ICahn Institute</a>
  <ul class="collapse">
  <li><a href="#towfique-david.-review-of-aims" id="toc-towfique-david.-review-of-aims" class="nav-link" data-scroll-target="#towfique-david.-review-of-aims">Towfique / David. Review of aims</a></li>
  <li><a href="#chirag.-tutorial-deep-learning-gene-regulation" id="toc-chirag.-tutorial-deep-learning-gene-regulation" class="nav-link" data-scroll-target="#chirag.-tutorial-deep-learning-gene-regulation">Chirag. Tutorial: Deep learning gene regulation</a></li>
  <li><a href="#kailash.-big-brain-updates" id="toc-kailash.-big-brain-updates" class="nav-link" data-scroll-target="#kailash.-big-brain-updates">Kailash. Big brain updates</a></li>
  <li><a href="#ariana.-overview-of-thesis-aims" id="toc-ariana.-overview-of-thesis-aims" class="nav-link" data-scroll-target="#ariana.-overview-of-thesis-aims">Ariana. Overview of thesis aims</a></li>
  <li><a href="#karin.-probabilistic-modeling-of-single-cell-splicing" id="toc-karin.-probabilistic-modeling-of-single-cell-splicing" class="nav-link" data-scroll-target="#karin.-probabilistic-modeling-of-single-cell-splicing">Karin. Probabilistic modeling of single-cell splicing</a></li>
  <li><a href="#aline-real.-bigbrain-pca-and-sqtls" id="toc-aline-real.-bigbrain-pca-and-sqtls" class="nav-link" data-scroll-target="#aline-real.-bigbrain-pca-and-sqtls">Aline Real. BigBrain PCA and sQTLs</a></li>
  <li><a href="#anjali.-deep-learning-for-rare-variants" id="toc-anjali.-deep-learning-for-rare-variants" class="nav-link" data-scroll-target="#anjali.-deep-learning-for-rare-variants">Anjali. Deep learning for rare variants</a></li>
  <li><a href="#tatsuhiko.-deep-learning-splicing-qtl-in-microglia" id="toc-tatsuhiko.-deep-learning-splicing-qtl-in-microglia" class="nav-link" data-scroll-target="#tatsuhiko.-deep-learning-splicing-qtl-in-microglia">Tatsuhiko. Deep learning splicing QTL in microglia</a></li>
  <li><a href="#winston.-rna-editing" id="toc-winston.-rna-editing" class="nav-link" data-scroll-target="#winston.-rna-editing">Winston. RNA editing</a></li>
  <li><a href="#beomjin-jang.-microglia-twas-singlebrain" id="toc-beomjin-jang.-microglia-twas-singlebrain" class="nav-link" data-scroll-target="#beomjin-jang.-microglia-twas-singlebrain">Beomjin Jang. Microglia TWAS / SingleBrain</a></li>
  <li><a href="#teresa-lin.-functional-finemapping-and-prs." id="toc-teresa-lin.-functional-finemapping-and-prs." class="nav-link" data-scroll-target="#teresa-lin.-functional-finemapping-and-prs.">Teresa Lin. Functional finemapping and PRS.</a></li>
  <li><a href="#ashvin-ravi.-meta-myeloid" id="toc-ashvin-ravi.-meta-myeloid" class="nav-link" data-scroll-target="#ashvin-ravi.-meta-myeloid">Ashvin Ravi. Meta-myeloid</a></li>
  </ul></li>
  </ul>
<div class="toc-actions"><div><i class="bi bi-github"></i></div><div class="action-links"><p><a href="https://github.com/banskt/nygc-notes/edit/main/listen/group-meeting/2023-04-14-adsp-group-meeting.qmd" class="toc-action">Edit this page</a></p><p><a href="https://github.com/banskt/nygc-notes/issues/new" class="toc-action">Report an issue</a></p></div></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">2023-04-14</h1>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<section id="adsp-group-meeting-at-icahn-institute" class="level2">
<h2 class="anchored" data-anchor-id="adsp-group-meeting-at-icahn-institute">ADSP Group Meeting at ICahn Institute</h2>
<p>Knowles Lab and Raj Lab (Towfique Raj) works together on the Alzheimer’s Disease Sequencing Project (ADSP). Here are my notes on the in-person joint meeting.</p>
<section id="towfique-david.-review-of-aims" class="level3">
<h3 class="anchored" data-anchor-id="towfique-david.-review-of-aims">Towfique / David. Review of aims</h3>
<p>17k WGS, 40 cohorts, mix of ancestry. Find new genes, alleles, etc for AD.</p>
<ul>
<li>Functional genomics. xQTL (Gao Wang and Phil de Jager). Multi-omic functional assessment of novel AD variants using single-cell (Montine, Kundaje, Montgomery).</li>
<li>AI / ML: neuro-imaging. Olivier Lichtarge - cognitive computing of AD genes and risk. Wein Pan - causal and integrative dep learning for AD genetics. Paul Thompson - mega-grant.</li>
<li>Structural variant WG.</li>
</ul>
<section id="reguloml" class="level4">
<h4 class="anchored" data-anchor-id="reguloml">ReguloML</h4>
<p>Apply deep learning techniques to predict and interpret the functional effects of AD variants, gene regulation, gene networks (RNA-Seq, ATAC-Seq, cell-type specific), trans-eQTL, prioritization of genes and pathways (finemapping?).</p>
</section>
</section>
<section id="chirag.-tutorial-deep-learning-gene-regulation" class="level3">
<h3 class="anchored" data-anchor-id="chirag.-tutorial-deep-learning-gene-regulation">Chirag. Tutorial: Deep learning gene regulation</h3>
<p>Training functional genomics models in PyTorch.</p>
<p>DL models predict functional genomic assays as a function of sequence. DNA -&gt; Convolution Layers -&gt; Dilated convolution layers -&gt; Prediction.</p>
<p>Kelley et al, Genome Research, 2018.</p>
<p>For genomics, the goal is two-fold: (1) classification - predict called peaks for a particular assay, (2) regression - predict read counts of that assay.</p>
<p>Bias correction is important. read counts - assay bias. Basenji - GC bias. BPNet - bias correction model. How do this bias correction methods work?</p>
<p>Input format - one hot encoding of DNA sequence.</p>
<p>Output - generate bins of read counts for each functional genomics assay.</p>
<p>Create Dataloader - tiled matrix of DNA. Split data for training / validation / test.</p>
</section>
<section id="kailash.-big-brain-updates" class="level3">
<h3 class="anchored" data-anchor-id="kailash.-big-brain-updates">Kailash. Big brain updates</h3>
<p>Cell-type specific distal genetic regulation.</p>
<p><em>Goal</em>: cis and trans eQTLs for a massive resource of uniformly processed RNA-Seq and genotype data (BigBrain) - expression, isoforms and splicing.</p>
<p>There are several cohorts and the team has calculated summary statistics for cis-eQTLs in most of them.</p>
<p>For one particular cohort, they found very few sQTLs. Is there someting wrong with the ROSMAP cohort? One potential source of error could be because ROSMAP samples are reverse-stranded.</p>
<p>They are planning to extend to trans-eQTLs.</p>
<p><a href="https://rajlabmssm.github.io/BigBrain">Link</a></p>
<p>Chirag asked about isoform TWAS.</p>
</section>
<section id="ariana.-overview-of-thesis-aims" class="level3">
<h3 class="anchored" data-anchor-id="ariana.-overview-of-thesis-aims">Ariana. Overview of thesis aims</h3>
<p>Impact of genetic variants on TF binding and chromatin in AD in cell-type specific ways.</p>
<p>GWAS identified 75+ AD associated genetic loci. Myeloid cells are important.</p>
<p>AD risk loci mediated by gene expression in microglia. MiGA (Microglia Genomic Atlas). Paiva Lopes, … Raj, Nat Genet.</p>
<p>caQTLs - chromatin accessibility.</p>
<p>bQTLs - TF binding QTLs, from novel pooled ChIP-Seq.</p>
<p>Harmonize bulk, sorted and single-cell/nuclei ATAC-seq datasets from neuronal and non-neuronal cells, enriching for myeloid cells, create chromatin accesibility profiles. About 1063 total donors bulk, 179 total donors in single nuclei.</p>
<p>How myeloid chromatin states compare to other cell types?</p>
<p>Geschwind lab - ATAC-seq bulk - fetal brains, PHNPC cell types, 92 donors - NPD</p>
<p>Monocytes are good proxy for microglia.</p>
<p>snMultiomics - RNA, ATAC. Also genotyped.</p>
</section>
<section id="karin.-probabilistic-modeling-of-single-cell-splicing" class="level3">
<h3 class="anchored" data-anchor-id="karin.-probabilistic-modeling-of-single-cell-splicing">Karin. Probabilistic modeling of single-cell splicing</h3>
<p>Evolutionary conservation of alternative splicing. Generally AS is quite species specific. Mazin et al, Nat Genet. 2021 showed three cassette exons in DLG3 that are highly conserved AS events in the brain. But, we know little about splicing conservation in single cells.</p>
<p>How well can we measure AS in single cell? Olivieri et al, eLife 2021, Hardwick et al, Nat Biotechnol, 2022</p>
<p>SPLiT-Seq with PolyDT + random hexamer primers provide better coverage than 10X genomics with polyDT primers.</p>
<p>ZooBrain: regulatory conservation across vertebrate brains. Satija group. How to define splicing events from RNA-Seq data?</p>
<p>Are junction counts zero inflated? 39% of junctions show zero-inflation under a global binomial model. Whereas only ~1% show zero inflation with individual binomial model.</p>
<p>Do PBMCs undergo differential splicing?</p>
<p>Karin is also looking at topic-modeling of single-cells.</p>
</section>
<section id="aline-real.-bigbrain-pca-and-sqtls" class="level3">
<h3 class="anchored" data-anchor-id="aline-real.-bigbrain-pca-and-sqtls">Aline Real. BigBrain PCA and sQTLs</h3>
<p>Assessing the effect of different variables on gene expression and define appropriate covariates for sQTL analysis.</p>
<p>splicing QTLs are variants that modify alternative splicing events. She is using 9 cohorts (n ~ 5.5k).</p>
<p>First 3 PCs of the gene expression explain more than 50% variation. She thinks that cohort and tissue types are the major driver of the variation. Needs to be accounted for in the sQTL analysis. She also checked for library preparation as a possible covariate, she checked for number of reads and sex as covariates.</p>
<p>Can we try using any unsupervised method for predicting the hidden covariates?</p>
<p>Identify a good approach to account for covariate effects while performing sQTL analysis.</p>
</section>
<section id="anjali.-deep-learning-for-rare-variants" class="level3">
<h3 class="anchored" data-anchor-id="anjali.-deep-learning-for-rare-variants">Anjali. Deep learning for rare variants</h3>
<p>Improving rare variant association tests using functional annotations.</p>
<p>She talked about Functional Score Test (FST). We expect the FST to capture the signature of functional annotation The FST does not appear to be capturing the difference between known AD genes and random genes. Also see <a href="../../listen/group-meeting/2023-04-12-adas-group-meeting.html">my notes</a> on her group meeting earlier this week.</p>
</section>
<section id="tatsuhiko.-deep-learning-splicing-qtl-in-microglia" class="level3">
<h3 class="anchored" data-anchor-id="tatsuhiko.-deep-learning-splicing-qtl-in-microglia">Tatsuhiko. Deep learning splicing QTL in microglia</h3>
<p>Why microglia? Involved in A<span class="math inline">\(\beta\)</span> plaque formation.</p>
<p>The project aims to contruct prediction model. Training: Brain (microglia) RNA-Seq. Learn DL model and then predict.</p>
<p>Pangolin - improving multiple tissue prediction by combining different species. It is trained on 4 species, human rhesus, mouse and rat. 10,000 base pair sequence context, dilated residual convolution layers, and predict splice probability. Pangolin outperforms SpliceAI.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../images/pangolin_01.png" class="img-fluid figure-img" style="width:80.0%"></p>
<figcaption class="figure-caption">Pangolin</figcaption>
</figure>
</div>
<p>Tatsuhiko showed the pipeline for generating training and test data for Pangolin. Essentially, the idea is to find the splice sites and find association with reference genome. He will use microglia data from other species, published elsewhere recently. He discussed a detailed benchmarking plan for comparison of his new microglia data training with the origin training. The expectation is to improve accuracy with increasing sample size and species.</p>
</section>
<section id="winston.-rna-editing" class="level3">
<h3 class="anchored" data-anchor-id="winston.-rna-editing">Winston. RNA editing</h3>
<p>A-to-I RNA editing is abundant in the brain and implicated in disease.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><img src="../../images/rna-editing.jpg" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">RNA editing</figcaption>
</figure>
</div>
<section id="bigbrain" class="level4">
<h4 class="anchored" data-anchor-id="bigbrain">BigBrain</h4>
<p>n = 11801 (!)</p>
<p>Methods - RNA editing events called with Jacusa2. edQTL mapping with TensorQTL.</p>
<p>Meta-analysis drastically improves discovery revealing 21620 edQTLs in 5866 genes. But, he is observing many “singleton” sites, that is, associations observed in single cohorts. edSites occur in hyper-editing clusters.</p>
<p>Colocalization analysis.</p>
<p>edQTLs colocalize with Parkinson’s disease risk SNPs.</p>
</section>
</section>
<section id="beomjin-jang.-microglia-twas-singlebrain" class="level3">
<h3 class="anchored" data-anchor-id="beomjin-jang.-microglia-twas-singlebrain">Beomjin Jang. Microglia TWAS / SingleBrain</h3>
</section>
<section id="teresa-lin.-functional-finemapping-and-prs." class="level3">
<h3 class="anchored" data-anchor-id="teresa-lin.-functional-finemapping-and-prs.">Teresa Lin. Functional finemapping and PRS.</h3>
<p>Pipeline using Wightman summary stat, QC, functional fine-mapping (PolyFun) and statistical fine-mapping (SuSiE), finally Polygenic Risk Score (Plink with clumping + p threshold).</p>
<p>Teresa is comparing the accuracy of finemapping by looking at the heritability enrichment obtained from the finemapped variants, as well as using PRS performance of the finemapped variants.</p>
<p>Question: Is the PRS score measured on separate test data?</p>
</section>
<section id="ashvin-ravi.-meta-myeloid" class="level3">
<h3 class="anchored" data-anchor-id="ashvin-ravi.-meta-myeloid">Ashvin Ravi. Meta-myeloid</h3>
<p>Meta-analysis of myeloid cell eQTLs.</p>
<p>Use the myeloid cells from the BigBrain project.</p>
<p>Many eQTLs colocalize with AD GWAS loci.</p>
<p>He wants to develop framework to understand enrichment for eQTL (and sQTL) data. - Approach 1: fine map and use results as continuous annotation, assess heritability via LD score regression. - Approach 2: run mediated expression score regression (MESC).</p>


</section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../../listen/group-meeting/2023-04-12-adas-group-meeting.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">2023-04-12</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../../listen/group-meeting/2023-04-19-tatsuhiko-naito.html" class="pagination-link">
        <span class="nav-page-text">2023-04-19</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav>
</div> <!-- /content -->



</body></html>